Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/RuntimeWrapper.cs

```diff
@@ -43,5 +43,17 @@ public object ReadFile(string fileName)
                 return null;
             }
         }
+
+        [UsedImplicitly]
+        public object GetWorldTime()
+        {
+            try { return api.GetWorldTimeAsync(cancellationTokenSource.Token).AsTask().ToPromise(); }
+            catch (Exception e)
+            {
+                // Report an uncategorized exception
+                exceptionsHandler.OnEngineException(e);
+                return null;
```

Comment:
Did you try this flow in js context? What happens with the scene flow? Perhaps, instead of returning an invalid promise, you could return a promise with an empty object?

Reply:
I did not try this flow tbh. Im thinking that I could move the try catch to the RuntimeImplementation directly, similar to what's done in the EtheremApiWrapper.  So if anything goes wrong during GetWorldTimeAsync, we still return a valid response with a default time.

Reply:
I dont think that returning a fallback value is desired tho, because it may provoke inconsistencies in the scene without noticing. I think it should become invalid in js in case of exception, thats why i think that you should return just an empty object or something similar so you can make the check for its validity in js context

Reply:
Okay, you are right in that :D
So, I made changes to the return, but please let me know if Im doing it horribly, because I never used C# to talk to js so for now its a bit of a black box for me xD
Also changed the js code to handle the issue. Again, let me know if you think it is a good solution. TBH I havent seen any other of these APIs handle errors in any way xD so Im not sure how we would like to do it. If return an error or just do an debug error.
On the other hand, I used this -> `Task.FromException(e).ToPromise()` which seemed appropriate, but again, Im not sure how the js side will like it. The other option I was thinking was doing something like `Task.FromResult(new object()).ToPromise()`. 
Do you have another idea?
Thank you 🙏

Reply:
Yes i think `Task.FromException(e).ToPromise()` is great! You can test how it works by forcing the flow so always throws an exception